crates/nu-command/src/database/values/dsl/expression.rs

```diff
@@ -112,6 +112,7 @@ impl CustomValue for ExprDb {
             Operator::Multiply => Ok(BinaryOperator::Multiply),
             Operator::Divide => Ok(BinaryOperator::Divide),
             Operator::Modulo => Ok(BinaryOperator::Modulo),
+            Operator::IntegerDivision => Ok(BinaryOperator::Divide),
```

Comment:
If we need `Operator::IntegerDivision`, I'd suggest rename it to `Operator::FloorDivision`